src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -16,30 +16,138 @@
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import { getFormattedLocation } from 'src/shared/helpers/formatters/regionFormatter';
+import { getStrandDisplayName } from 'src/shared/helpers/formatters/strandFormatter';
+import {
+  buildFocusIdForUrl,
+  getDisplayStableId
+} from 'src/shared/state/ens-object/ensObjectHelpers';
 import { getBrowserActiveEnsObject } from 'src/content/app/browser/browserSelectors';
 
+import ViewInApp from 'src/shared/components/view-in-app/ViewInApp';
+
 import { EnsObjectGene } from 'src/shared/state/ens-object/ensObjectTypes';
+import { Gene as GeneFromGraphql } from 'src/shared/types/thoas/gene';
+
+import styles from './GeneSummary.scss';
+
+const GENE_QUERY = gql`
+  query Gene($genomeId: String!, $geneId: String!) {
+    gene(byId: { genome_id: $genomeId, stable_id: $geneId }) {
+      alternative_symbols
+      name
+      stable_id
+      symbol
+      so_term
+      transcripts {
+        stable_id
+      }
+      slice {
+        strand {
+          code
+          value
+        }
+        location {
+          length
+        }
+      }
+    }
+  }
+`;
 
-import styles from 'src/content/app/browser/drawer/Drawer.scss';
+type Gene = Required<
+  Pick<
+    GeneFromGraphql,
+    | 'stable_id'
+    | 'symbol'
+    | 'name'
+    | 'alternative_symbols'
+    | 'so_term'
+    | 'transcripts'
+    | 'slice'
+  >
+>;
 
 const GeneSummary = () => {
-  const ensObject = useSelector(getBrowserActiveEnsObject) as EnsObjectGene;
+  const ensObjectGene = useSelector(getBrowserActiveEnsObject) as EnsObjectGene;
+
+  const { data, loading } = useQuery<{ gene: Gene }>(GENE_QUERY, {
+    variables: {
+      geneId: ensObjectGene.stable_id,
+      genomeId: ensObjectGene.genome_id
+    },
+    skip: !ensObjectGene.stable_id
+  });
+
+  if (loading || !data?.gene) {
+    return null;
+  }
+
+  if (!data?.gene) {
+    return <div>No data available</div>;
+  }
+
+  const { gene } = data;
+
+  const stableId = getDisplayStableId(gene);
+
+  const focusId = buildFocusIdForUrl({
+    type: 'gene',
+    objectId: gene.stable_id as string
+  });
+  const entityViewerUrl = urlFor.entityViewer({
+    genomeId: ensObjectGene.genome_id,
+    entityId: focusId
+  });
 
   return (
-    <div className={styles.drawerView}>
-      <div className={styles.container}>
+    <div className={styles.container}>
+      <div className={styles.standardLabelValue}>
         <div className={styles.label}>Gene</div>
-        <div className={styles.details}>
-          <span className={styles.mainDetail}>{ensObject.label}</span>
+        <div className={styles.value}>
+          <div className={styles.featureDetails}>
+            {gene.symbol && (
+              <span className={styles.featureSymbol}>{gene.symbol}</span>
+            )}
+            {gene.symbol !== stableId && (
+              <span className={styles.stableId}>{stableId}</span>
+            )}
+            {gene.so_term && <div>{gene.so_term.toLowerCase()}</div>}
```

Comment:
thoas promises that there will always be an so_term on the gene. If there isn't, thoas will return an error.

Reply:
Removed the check